tests/components/admin_console/manage_teams_modal/manage_teams_modal.test.jsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ManageTeamsModal from 'components/admin_console/manage_teams_modal/manage_teams_modal.jsx';
+
+describe('ManageTeamsModal', () => {
+    function emptyFunc() {} // eslint-disable-line no-empty-function
+
+    const baseProps = {
+        onModalDismissed: emptyFunc,
+        show: true,
+        user: {
+            id: 'currentUserId',
+            last_picture_update: '1234',
+            email: 'currentUser@test.com',
+            roles: 'system_user',
+            username: 'currentUsername',
+        },
+        updateTeamMemberSchemeRoles: emptyFunc,
+        getTeamMembersForUser: emptyFunc,
+        getTeamsForUser: emptyFunc,
+    };
+
+    test('should match snapshot init', () => {
```

Comment:
Maybe split this test in two? It would be nice to separately test that the mount behaviour is correct from testing that it renders correctly

Reply:
Done.